web/client/codechecker_client/cmd_line_client.py

```diff
@@ -87,31 +87,35 @@ def run_sort_type_str(value: ttypes.RunSortType) -> Optional[str]:
     """ Converts the given run sort type to string. """
     if value == ttypes.RunSortType.NAME:
         return 'name'
-    elif value == ttypes.RunSortType.UNRESOLVED_REPORTS:
+    if value == ttypes.RunSortType.UNRESOLVED_REPORTS:
         return 'unresolved_reports'
-    elif value == ttypes.RunSortType.DATE:
+    if value == ttypes.RunSortType.DATE:
         return 'date'
-    elif value == ttypes.RunSortType.DURATION:
+    if value == ttypes.RunSortType.DURATION:
         return 'duration'
-    elif value == ttypes.RunSortType.CC_VERSION:
+    if value == ttypes.RunSortType.CC_VERSION:
```

Comment:
We should allow the elif statement here.

Reply:
Same answer.